Mathlib/CategoryTheory/Monad/Adjunction.lean

```diff
@@ -88,6 +88,62 @@ def adjToComonadIso (G : Comonad C) : G.adj.toComonad ≅ G :=
   ComonadIso.mk (NatIso.ofComponents fun X => Iso.refl _)
 #align category_theory.adjunction.adj_to_comonad_iso CategoryTheory.Adjunction.adjToComonadIso
 
+/--
+Given an adjunction `L ⊣ R`, if `L ⋙ R` is abstractly isomorphic to the identity functor, then the
+unit is an isomorphism.
+-/
+def unitAsIsoOfAbstractIso (adj : L ⊣ R) (i : L ⋙ R ≅ 𝟭 C) :  𝟭 C ≅ L ⋙ R where
```

Comment:
Let's remove `Abstract` from the name, here and below, so just `unitAsIsoOfIso`.